mdsuite/graph_modules/molecular_graph.py

```diff
@@ -217,8 +225,9 @@ def check_a_in_b(a, b):
             return False
 
         molecules = {}
+        log.info(f"Building molecular graph from configuration for {self.molecule_name}")
         # TODO speed up
-        for i in tqdm(range(len(adjacency_matrix)), desc="Building molecules"):
+        for i in tqdm(range(len(adjacency_matrix)), ncols=70):
             indices = tf.where(adjacency_matrix[i])
             indices = tf.reshape(indices, (len(indices)))
```

Comment:
Is this equivalent to `tf.reshape(indices, -1)` ?

Reply:
resolved 5a142bc